### ISSUE TITLE
Missing 'has[2]' in the `read_door` function?

### ISSUE THREAD

**Issue by qixuema** [sepidsh/Housegan-data-reader#8]
Hi @sepidsh 

I was reviewing the code in the repository and noticed that there seems to be a missing 'has[2]' variable in the code.

Could you please confirm if this is an oversight or intentional omission? If it is indeed an oversight, I would like to suggest adding the 'has[2]' variable to the code in order to ensure its completeness and functionality.

Thank you for your attention to this matter. I look forward to your response.

Best regards,
Xueqi

**Comment by sepidsh** [comment]
Hi, 
The code at correct form has been tested and the current form has been used for HouseGAN++. 

Thank you
Regards
Sepid

**Comment by qixuema** [comment]
@sepidsh 
Yes, the code has indeed been tested, but I still don't understand why `has[2]` is missing here. 🤔
